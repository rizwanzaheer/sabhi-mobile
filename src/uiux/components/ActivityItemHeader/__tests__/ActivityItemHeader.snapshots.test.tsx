import * as React from 'react'
import { render } from 'react-native-testing-library'
import ActivityHeader from '../ActivityItemHeader'

describe('Component(snapshots): ActivityHeader', () => {
  const action = jest.fn()
  const viewer = {
    did: 'ethr:did:123456789',
    shortId: 'Test Viewer',
    profileImage: 'http://',
  }
  const baseProps = {
    id: 'ZFGHFSJD',
    date: 123445678910,
    issuer: {
      name: 'Test Issuer',
      did: 'ethr:did:123456',
      avatar: { uri: 'http://' },
      shortId: 'Test Issuer',
    },
    subject: {
      name: 'Test Subject',
      did: 'ethr:did:123456',
      avatar: { uri: 'http://' },
      shortId: 'Test Name',
    },
    profileAction: action,
  }

  it('should render incoming true with reason', () => {
    const tree = render(
      <ActivityHeader viewer={viewer} reason={'to test the components'} {...baseProps} />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should render incoming false with reason', () => {
    const tree = render(
      <ActivityHeader viewer={viewer} reason={'to test the components'} {...baseProps} />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should render incoming true without reason', () => {
    const tree = render(<ActivityHeader viewer={viewer} {...baseProps} />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should render incoming false without reason', () => {
    const tree = render(<ActivityHeader viewer={viewer} {...baseProps} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
