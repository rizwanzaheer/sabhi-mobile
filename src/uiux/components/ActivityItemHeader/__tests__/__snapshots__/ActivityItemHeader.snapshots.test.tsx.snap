// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component(snapshots): ActivityHeader should render incoming false with reason 1`] = `
<View
  pointerEvents="auto"
  style={
    Object {
      "alignItems": undefined,
      "backgroundColor": undefined,
      "borderRadius": undefined,
      "flex": undefined,
      "flexDirection": undefined,
      "height": undefined,
      "justifyContent": undefined,
      "margin": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "padding": undefined,
      "paddingBottom": undefined,
      "paddingHorizontal": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
      "width": undefined,
    }
  }
>
  <View
    pointerEvents="auto"
    style={
      Object {
        "alignItems": undefined,
        "backgroundColor": undefined,
        "borderRadius": undefined,
        "flex": 1,
        "flexDirection": undefined,
        "height": undefined,
        "justifyContent": undefined,
        "margin": undefined,
        "marginBottom": 4,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginTop": undefined,
        "padding": undefined,
        "paddingBottom": undefined,
        "paddingHorizontal": undefined,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingTop": undefined,
        "width": undefined,
      }
    }
  >
    <Text
      style={
        Object {
          "flexWrap": "wrap",
        }
      }
    >
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
        Test Issuer
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
          }
        }
      >
         
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
          }
        }
      >
         so 
        Test Name
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
         
        to test the components
      </Text>
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#BBBBBB",
        "flexWrap": "wrap",
        "fontSize": 15,
      }
    }
  >
    over 46 years
     ago
  </Text>
</View>
`;

exports[`Component(snapshots): ActivityHeader should render incoming false without reason 1`] = `
<View
  pointerEvents="auto"
  style={
    Object {
      "alignItems": undefined,
      "backgroundColor": undefined,
      "borderRadius": undefined,
      "flex": undefined,
      "flexDirection": undefined,
      "height": undefined,
      "justifyContent": undefined,
      "margin": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "padding": undefined,
      "paddingBottom": undefined,
      "paddingHorizontal": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
      "width": undefined,
    }
  }
>
  <View
    pointerEvents="auto"
    style={
      Object {
        "alignItems": undefined,
        "backgroundColor": undefined,
        "borderRadius": undefined,
        "flex": 1,
        "flexDirection": undefined,
        "height": undefined,
        "justifyContent": undefined,
        "margin": undefined,
        "marginBottom": 4,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginTop": undefined,
        "padding": undefined,
        "paddingBottom": undefined,
        "paddingHorizontal": undefined,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingTop": undefined,
        "width": undefined,
      }
    }
  >
    <Text
      style={
        Object {
          "flexWrap": "wrap",
        }
      }
    >
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
        Test Issuer
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
          }
        }
      >
         
      </Text>
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
         
        Test Name
      </Text>
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#BBBBBB",
        "flexWrap": "wrap",
        "fontSize": 15,
      }
    }
  >
    over 46 years
     ago
  </Text>
</View>
`;

exports[`Component(snapshots): ActivityHeader should render incoming true with reason 1`] = `
<View
  pointerEvents="auto"
  style={
    Object {
      "alignItems": undefined,
      "backgroundColor": undefined,
      "borderRadius": undefined,
      "flex": undefined,
      "flexDirection": undefined,
      "height": undefined,
      "justifyContent": undefined,
      "margin": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "padding": undefined,
      "paddingBottom": undefined,
      "paddingHorizontal": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
      "width": undefined,
    }
  }
>
  <View
    pointerEvents="auto"
    style={
      Object {
        "alignItems": undefined,
        "backgroundColor": undefined,
        "borderRadius": undefined,
        "flex": 1,
        "flexDirection": undefined,
        "height": undefined,
        "justifyContent": undefined,
        "margin": undefined,
        "marginBottom": 4,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginTop": undefined,
        "padding": undefined,
        "paddingBottom": undefined,
        "paddingHorizontal": undefined,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingTop": undefined,
        "width": undefined,
      }
    }
  >
    <Text
      style={
        Object {
          "flexWrap": "wrap",
        }
      }
    >
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
        Test Issuer
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
          }
        }
      >
         
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
          }
        }
      >
         so 
        Test Name
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
         
        to test the components
      </Text>
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#BBBBBB",
        "flexWrap": "wrap",
        "fontSize": 15,
      }
    }
  >
    over 46 years
     ago
  </Text>
</View>
`;

exports[`Component(snapshots): ActivityHeader should render incoming true without reason 1`] = `
<View
  pointerEvents="auto"
  style={
    Object {
      "alignItems": undefined,
      "backgroundColor": undefined,
      "borderRadius": undefined,
      "flex": undefined,
      "flexDirection": undefined,
      "height": undefined,
      "justifyContent": undefined,
      "margin": undefined,
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "padding": undefined,
      "paddingBottom": undefined,
      "paddingHorizontal": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
      "width": undefined,
    }
  }
>
  <View
    pointerEvents="auto"
    style={
      Object {
        "alignItems": undefined,
        "backgroundColor": undefined,
        "borderRadius": undefined,
        "flex": 1,
        "flexDirection": undefined,
        "height": undefined,
        "justifyContent": undefined,
        "margin": undefined,
        "marginBottom": 4,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginTop": undefined,
        "padding": undefined,
        "paddingBottom": undefined,
        "paddingHorizontal": undefined,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingTop": undefined,
        "width": undefined,
      }
    }
  >
    <Text
      style={
        Object {
          "flexWrap": "wrap",
        }
      }
    >
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
        Test Issuer
      </Text>
      <Text
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
          }
        }
      >
         
      </Text>
      <Text
        onPress={[Function]}
        style={
          Object {
            "color": "#333333",
            "flexWrap": "wrap",
            "fontSize": 17,
            "fontWeight": "bold",
          }
        }
      >
         
        Test Name
      </Text>
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#BBBBBB",
        "flexWrap": "wrap",
        "fontSize": 15,
      }
    }
  >
    over 46 years
     ago
  </Text>
</View>
`;
